### DISCUSSION TITLE
Is TrguiNG portable?

### DISCUSSION THREAD

**Discussion by WirlyWirly** [openscopeproject/TrguiNG#242] [Q&A]
Hello,

I just found TrguiNG and I'm liking it a lot, however I was wondering how portable the application is; Will it leave behind traces if I'm using it from a USB drive?

I don't see a .zip release, only a .exe/.msi, so I take it this program isn't portable.

Is that something that can easily be accomplished akin to PortableApps? (leave no trace)

**Comment by qu1ck** [comment]
Windows version of TrguiNG is kinda portable, as long as you use it on windows machines with preinstalled webview2 component (all windows 11 machines and some windows 10). But it does leave traces in local appdata.

I am saying kinda portable because it expects to find the config in `%appdata%\trguing.json`. Also the webview leaves it's cache and storage data in `%localappdata%\org.openscopeproject.trguing`.
If you write a wrapper that will copy the config to the right place before launch and move it back after the app exits, as well as cleans up the localappdata folder, the app will be truly portable.

But much easier "portable" option is to use the web version of TrguiNG on your server instead, that way you only need a browser.
